src/api/test/functional/channel_maintenance_test.rb

```diff
@@ -790,6 +790,31 @@ def test_large_channel_test
     assert_xml_tag tag: 'binary', attributes:            { name: 'package', project: "BaseDistro3Channel", repository: "channel_repo", arch: "src" }
     assert_xml_tag tag: 'updateinfo', attributes:            { id: "UpdateInfoTagNew-patch_name-#{Time.now.utc.year}-1", version: "1" }
 
+    # drop and re-release manual release and try again
```

Comment:
At some point we have to stop adding lines to this test. This is really getting out of hand IMHO...